include/cocaine/rpc/asio/header.hpp

```diff
@@ -596,14 +649,17 @@ http2_integer_size(size_t sz, size_t bit_offset) {
 // Note dest should be at least 10 bytes long
 // See packing here https://httpwg.github.io/specs/rfc7541.html#integer.representation
 size_t
-http2_integer_encode(char* dest, size_t source, size_t bit_offset, char prefix) { 
+http2_integer_encode(unsigned char* dest, size_t source, size_t bit_offset, char prefix) {
```

Comment:
Because we do some ariphmetics which can overflow signed char. If I remember correctly that will practically always work, but it's UB in fact.